### PR TITLE
[Cache] Fix calculate ttl in couchbase sdk 3.0

### LINKED ISSUE #44049: [Cache] Failure with couchbase collection
### Symfony version(s) affected

5.4

### Description

In the CI we see a warning:
https://github.com/symfony/symfony/runs/4204456225#step:11:28

> [cb,WARN] (pcbc/pool L:37) The specified expiry duration 1668530562 is longer than 50 years. For bug-compatibility with previous versions of SDK 3.0.x, the number of seconds in the duration will be interpreted as the epoch second when the document should expire (#{effective_expiry}). Stuffing an epoch second into a Duration is deprecated and will no longer work in SDK 3.1. Consider using Time instance instead.

And a failure:
> 1) Symfony\Component\Cache\Tests\Adapter\CouchbaseCollectionAdapterTest::testExpiration
Failed asserting that true is false.
> src/Symfony/Component/Cache/Tests/Adapter/AdapterTestCase.php:169

And a warning:
> THE ERROR HANDLER HAS CHANGED!

To be investigated.

@ajcerezo since you contributed support for couchbase collection, maybe you'd like to have a look?

### How to reproduce

-

### Possible Solution

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by ajcerezo**
| Q             | A
| ------------- | ---
| Branch?       | 5.4 for features / 4.4 or 5.3 for bug fixes 
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no 
| Tickets       | Fix #44049
| License       | MIT
| Doc PR        |

### CODE REVIEW

**Review by stof — APPROVED** [review]

File: src/Symfony/Component/Cache/Adapter/CouchbaseCollectionAdapter.php [hunk 1]
```diff
@@ -221,13 +220,4 @@ protected function doSave(array $values, $lifetime)
 
         return [] === $ko ? true : $ko;
     }
-
-    private function normalizeExpiry(int $expiry): int
-    {
-        if ($expiry && $expiry > static::THIRTY_DAYS_IN_SECONDS) {
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Cache/Adapter/CouchbaseCollectionAdapter.php:227]: the const can be removed from the class declaration, isn't it?

**Review by nicolas-grekas — COMMENTED** [review]
looks like there is a failure, can you please have a look?

File: src/Symfony/Component/Cache/Adapter/CouchbaseCollectionAdapter.php (hunk 1 above)
Comment (ajcerezo) [reply src/Symfony/Component/Cache/Adapter/CouchbaseCollectionAdapter.php:227]: Yes, I do it

Comment (ajcerezo) [reply src/Symfony/Component/Cache/Adapter/CouchbaseCollectionAdapter.php:227]: This problem with `TestExpiration` only fails with couchbase `sdk 3.0.4` in `sdk 3.0.5` is not given. I think limiting compatibility in `sdk> = 3.0.5` is the correct solution

**Review by nicolas-grekas — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @ajcerezo.

**Pull request merged**